cogs/autorole.py

```diff
@@ -551,7 +551,11 @@ async def promote_to_friend(self, ctx, *users: str):
                 'ghost-proxy-friend'
             ],
             users,
-            role_limits=['ghost-proxy-member']
+            role_limits=[
+                'ghost-proxy-member',
+                'ghost-proxy-legacy',
+                'ghost-proxy-envoy'
+            ]
```

Comment:
Good catch, I don't think we need this for standard procedure but makes mistakes easier to fix.